src/landsat_importer/main.py

```diff
@@ -52,7 +52,7 @@ def main():
     parser.add_argument("input_path",help="Specify the path to the input landsat scene, may be a folder or metadata filename")
     parser.add_argument("output_path", help="Specify the output folder or filename")
 
-    parser.add_argument("--bands", help="Provide a comma separated list of the bands to export", default="")
+    parser.add_argument("--bands", nargs="+", help="Provide a comma separated list of the bands to import")
```

Comment:
I don't think the list is comma separated anymore if you're using nargs > 1

Reply:
well spotted!